hsa-amd-aqlprofile/PKGBUILD

```diff
@@ -1,30 +1,23 @@
 # Maintainer: Ilya Elenskiy <elenskiy.ilya@gmail.com>
+# Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
+
 pkgname=hsa-amd-aqlprofile
 pkgver=1.0.0
-pkgrel=2
-_debfile=hsa-amd-aqlprofile_1.0.0_amd64.deb 
-pkgdesc="AQLPROFILE library for AMD HSA runtime API extension support"
+pkgrel=3
+_debfile="hsa-amd-aqlprofile_${pkgver}_amd64.deb"
+pkgdesc='AQLPROFILE library for AMD HSA runtime API extension support'
 arch=('x86_64')
-url="https://github.com/RadeonOpenCompute/HSA-AqlProfile-AMD-extension"
+url='https://github.com/RadeonOpenCompute/ROCR-Runtime'
```

Comment:
Isn't this the upstream: https://github.com/HSAFoundation/HSA-Profiler-AMD?

Reply:
Open the link and you'll see it's deprecated by another repo that is already deprecated and you end up [here](https://github.com/GPUOpen-Tools/radeon_compute_profiler)

Reply:
But the package itself is still from the original link tho, the other links are just what it is deprecated by.

Reply:
This library is used by some current version ones (hsa-rocr tries to load this). It would be better to know where this is compiled from.

Reply:
Yeah we should atleast put the url link where the source code resides for the deb package.